### ISSUE TITLE
Is it possible to change the base URL?

### ISSUE THREAD

**Issue by mprigge** [requests/httpbin#509]
Is there support for running requestbin on a mapped url (e.g. mapped via a reverse proxy from http://example.com/requestbin/ to the docker container's /)? It doesn't seem like it, but I just wanted to make sure I didn't miss something super obvious. 

Thank you!

**Comment by mprigge** [comment]
Yeah - I did try it and the front page (at least) does not render correctly. All of the self-referencing links for stylesheets and script content appear to be defined absolutely (so they're trying to load /blah.css when you're seeing it as /httpbin/blah.css). Actually, come to think of it, that's almost all swagger content. I bet the individual endpoints probably work fine. So, it may not be that it's _entirely_ broken.

**Comment by javabrett** [comment]
For S

**Comment by mprigge** [comment]
A little bit, I think? It seems as though swagger does infer some settings based upon how it's called (you call using https, it responds accordingly). It doesn't seem like that works for the base path, however. Perhaps [this](https://github.com/requests/httpbin/blob/bd3c030734b9891eafbbedcc4863191938d51eb0/httpbin/core.py#L111) would need to be changed to make it work "perfectly" when proxied in this way. I haven't pulled it to try to rebuild it myself to test that, but I'm guessing an environment variable tied to that would probably do the job.

Totally could be wrong, though - still noodling it. And this is also really not a big deal. The testing endpoints still work and it's not as if this is something everyone wants or needs to be able to do. It's helpful to me as a testing service to make sure ingress is working properly in a kubernetes environment and you don't need the UI to look perfect in order to use it for that.
